PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -236,7 +236,7 @@ private void RemoveScrollBars(ScintillaControl sender)
 
         #region Scintilla Main
 
-        public ScintillaControl() : this(IntPtr.Size == 4 ? "SciLexer.dll" : "SciLexer64.dll")
+        public ScintillaControl() : this(Path.Combine(PathHelper.AppDir, IntPtr.Size == 4 ? "SciLexer.dll" : "SciLexer64.dll"))
```

Comment:
Before we were loading Scintilla based on relative path, which depends on the current running path... this can lead to multiple problems... I'm surprised we didn't run into this before? While doing the migration to NUnit 3 I stumbled into this problem... but when going backwards to NUnit 2 I was still facing this problem.

This can also make FD to fail depending on how it is run.